hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/ContainerBalancer.java

```diff
@@ -93,6 +94,7 @@
   private Map<DatanodeDetails, ContainerMoveSelection> sourceToTargetMap;
   private Map<DatanodeDetails, Long> sizeLeavingNode;
   private Map<DatanodeDetails, Long> sizeEnteringNode;
+  private Map<DatanodeDetails, Long> remainingUsageBeforeOverUtilized;
```

Comment:
I think we should not add this map `remainingUsageBeforeOverUtilized`. We can probably use sizeLeavingNode here.